test/hex_tarball_SUITE.erl

```diff
@@ -21,7 +21,7 @@ memory_test(_Config) ->
       <<"build_tool">> => <<"rebar3">>
      },
     Contents = [{"src/foo.erl", <<"-module(foo).">>}],
-    {ok, {Tarball, Checksum}} = hex_tarball:create(Metadata, Contents),
+    {ok, {Tarball, Checksum, [{<<"maintainers">>,deprecated}]}} = hex_tarball:create(Metadata, Contents),
```

Comment:
could you test the warning in a separate example? The intention of `memory_test` and `disk_test` was to have minimal valid data and just test the memory and disk aspects.

Reply:
I haven't even put in tests explicitly for the warnings or errors... I only updated the tests so that they would pass. But can remove the maintainers field from this test.